templates/emails/transactional/noreply.text.jinja2

```diff
@@ -0,0 +1,7 @@
+Hi!
+
+    Thanks for getting in touch. This mailbox cannot receive any emails. If you need help, please go to https://app.simplelogin.io/dashboard/support to contact us.
```

Comment:
Nits: I think we can remove some empty spaces at the beginning of some lines here

Reply:
Fixed